agent/config/config.go

```diff
@@ -68,6 +68,10 @@ const (
 	// image cleanup.
 	DefaultNumImagesToDeletePerCycle = 5
 
+	// DefaultNumImagesToDeletePerCycle specifies the default number of nonecs containers to delete when agent performs
+	// nonecs containers cleanup.
```

Comment:
How about non ecs images? We do not have another var to define how many will be removed per cycle?

Reply:
We set this number  DefaultNumImagesToDeletePerCycle including both ecs and nonecs images, first clear ecs images and then nonecs images.